Test/dafny0/Twostate-Resolution.dfy.expect

```diff
@@ -753,19 +753,19 @@ Twostate-Resolution.dfy(445,20): Error: no label 'After' in scope at this time
 Twostate-Resolution.dfy(452,17): Error: an @-label can only be applied to a two-state function
 Twostate-Resolution.dfy(453,14): Error: an @-label can only be applied to a two-state lemma
 Twostate-Resolution.dfy(459,19): Error: an @-label can only be applied to a two-state function
-Twostate-Resolution.dfy(481,11): Error: expression is not allowed to invoke a lemma (reveal_OrdinaryOpaque)
+Twostate-Resolution.dfy(481,11): Error: to reveal a function (OrdinaryOpaque), append parentheses
```

Comment:
The test file (`Twostate-Resolution.dfy`) still has comments like `(admittedly, a poor error message)`. Please remove those now, since your error messages are good.